beacon-chain/core/blocks/exit.go

```diff
@@ -163,3 +164,43 @@ func verifyExitConditions(validator state.ReadOnlyValidator, currentSlot primiti
 	}
 	return nil
 }
+
+func ExitSignatureBatch(
```

Comment:
Maybe add some documentation of what this function does. I see the other functions even have the spec definition.

Reply:
This function shares a lot of code with `BLSChangesSignatureBatch` I do not see how to use generics to deduplicate it, but having the common parts as helpers may be useful.